Content/Items/Beach/Accessories.SeaglassLens.cs

```diff
@@ -0,0 +1,40 @@
+﻿using StarlightRiver.Content.Items.BaseTypes;
+using Terraria.ID;
+
+namespace StarlightRiver.Content.Items.Beach
+{
+	internal class SeaglassLens : SmartAccessory
+	{
+		public override string Texture => AssetDirectory.Assets + "Items/Beach/" + Name;
+
+		public SeaglassLens() : base("Seaglass Lens", "Your minions and sentries occasionally deal extra damage\nThis extra damage counts as a critical strike") { }
+
+		public override void Load()
+		{
+			StarlightProjectile.OnHitNPCEvent += BonusDamage;
+		}
+
+		public override void SafeSetDefaults()
+		{
+			Item.rare = ItemRarityID.Blue;
+			Item.value = Item.sellPrice(silver: 50);
+		}
+
+		private void BonusDamage(Projectile projectile, NPC target, NPC.HitInfo hit, int damageDone)
+		{
+			Player owner = Main.player[projectile.owner];
+
+			if (Equipped(owner) && projectile.DamageType == DamageClass.Summon && Main.rand.NextBool(10))
```

Comment:
check against local player being the owner since simpleStrikeNPC sends its own packets and if another acc / something else calls SetHitPacketStatus this will get called by other clients and server